db/migrate/20220126072957_create_salesforce_cases_tables.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class CreateSalesforceCasesTables < ActiveRecord::Migration[6.0]
+  def change
+    create_table :salesforce_cases do |t|
+      t.string :uid, null: true, index: true, unique: true
+      t.integer :topic_id, null: false, unique: true
+      t.string :contact_id, null: true
+      t.string :number
```

Comment:
This feels odd to me because the column is named number but we're storing it in a `string` column.

Reply:
In their API doc, they mentioned this field as `string` only. That's why I used it. Also, the `CaseNumber` field will include zero in the prefixes like `00001035`.